rules-reviewed/quarkus/springboot/springboot-webmvc-to-quarkus.windup.xml

```diff
@@ -16,22 +16,47 @@
     <rules>
         <rule id="springboot-webmvc-to-quarkus-00000">
             <when>
-                <file filename="{filenameParameter}-{*}.jar" />
+                <javaclass references="org.springframework.web.servlet.mvc.Controller">
```

Comment:
If you dont do a negative check on the quarkus dependency, this will fire even when they migrate to Quarkus using the quarkus extension

Reply:
Yep, you're absolutely right.
We discussed this with @PhilipCattanach and the idea is to then move this to check if the Quarkus extension is not there in a separate (Groovy?) rule.
For the time being we tried to avoid the situation of the analysis of the jar of a SB application using `spring-boot-starter-web` that hence has always a `spring-webmvc` JAR inside that would have triggered the previous `file` condition without being sure the application was really using anything about Spring MVC but always adding anyway a 13 SP mandatory issue that alerts about the code won't run on Quarkus which was true only if the application was using something from Spring MVC.

Reply:
that's right.... is RestController the one covered by Quarkus Spring extension